<launch>

    <arg name="pkg_name" value="apriltag_detection"/>
    <arg name="veh" default="csc22918"/>
    <arg name="int_path" default="/data/config/calibrations/camera_intrinsic/$(arg veh).yaml"/>
    <arg name="ext_path" default="/data/config/calibrations/camera_extrinsic/$(arg veh).yaml"/>

    <!-- <group ns="$(arg veh)"> -->
    <node  name="apriltag_detection_node" pkg="$(arg pkg_name)" type="apriltag_detection_node.py" output="screen">
        <param name="veh" value="$(arg veh)"/>
        <param name="int_path" value="$(arg int_path)"/>
        <param name="ext_path" value="$(arg ext_path)"/>
    </node>
    <!-- </group> -->

</launch>
